pkg/metadata/host/host_windows.go

```diff
@@ -9,6 +9,8 @@ import "github.com/shirou/gopsutil/host"
 
 type osVersion [2]string
 
+const osName = "win32"
```

Comment:
can you add a comment so future me will remember why we're doing this? 😄

Reply:
Yupp! Done 👍